src/dispatchers/mouseDispatcher.ts

```diff
@@ -45,109 +46,151 @@ export module Dispatchers {
+    public onMouseUp(callback: MouseCallback): Dispatchers.Mouse {
+      this._setCallback(this._upCallbackSet, callback);
+      return this;
+    }
+
+    /**
+     * Registers the callback to be called whenever a mouseup occurs,
+     *
+     * @param {(p: Point) => any} callback A callback that takes the pixel position
+     *                                     in svg-coordinate-space. Pass `null`
+     *                                     to remove a callback.
+     * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
+     */
+    public offMouseUp(callback: MouseCallback): Dispatchers.Mouse {
+      this._unsetCallback(this._upCallbackSet, callback);
       return this;
     }
 
     /**
      * Registers a callback to be called whenever a wheel occurs,
-     * or removes the callback if `null` is passed as the callback.
      *
-     * @param {any} key The key associated with the callback.
-     *                  Key uniqueness is determined by deep equality.
      * @param {MouseCallback} callback A callback that takes the pixel position
      *                                     in svg-coordinate-space.
      *                                     Pass `null` to remove a callback.
      * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
      */
-    public onWheel(key: any, callback: MouseCallback): Dispatchers.Mouse {
-      this._setCallback(this._wheelBroadcaster, key, callback);
+    public onWheel(callback: MouseCallback): Dispatchers.Mouse {
+      this._setCallback(this._wheelCallbackSet, callback);
+      return this;
+    }
+
+    /**
+     * Registers the callback to be called whenever a wheel occurs,
+     *
+     * @param {MouseCallback} callback A callback that takes the pixel position
+     *                                     in svg-coordinate-space.
+     *                                     Pass `null` to remove a callback.
+     * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
+     */
+    public offWheel(callback: MouseCallback): Dispatchers.Mouse {
+      this._unsetCallback(this._wheelCallbackSet, callback);
       return this;
     }
 
     /**
      * Registers a callback to be called whenever a dblClick occurs,
-     * or removes the callback if `null` is passed as the callback.
      *
-     * @param {any} key The key associated with the callback.
-     *                  Key uniqueness is determined by deep equality.
      * @param {MouseCallback} callback A callback that takes the pixel position
      *                                     in svg-coordinate-space.
      *                                     Pass `null` to remove a callback.
      * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
      */
-    public onDblClick(key: any, callback: MouseCallback): Dispatchers.Mouse {
-      this._setCallback(this._dblClickBroadcaster, key, callback);
+    public onDblClick(callback: MouseCallback): Dispatchers.Mouse {
```

Comment:
Dispatcher endpoints currently reflect the event they are listening to.  Not sure if we should change this endpoint name